<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="59020" started-at="2012-09-04T12:09:53Z" finished-at="2012-09-04T12:10:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="59020" started-at="2012-09-04T12:09:53Z" finished-at="2012-09-04T12:10:52Z">
      <class name="com.registration.tests.notWorking.RegistrationAllCitiesTest">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.registration.tests.notWorking.RegistrationAllCitiesTest@2bb8ee96]" name="setUp" is-config="true" duration-ms="26445" started-at="2012-09-04T16:09:53Z" finished-at="2012-09-04T16:10:19Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="registrationAllCitiesTest()[pri:0, instance:com.registration.tests.notWorking.RegistrationAllCitiesTest@2bb8ee96]" name="registrationAllCitiesTest" duration-ms="25549" started-at="2012-09-04T16:10:19Z" finished-at="2012-09-04T16:10:45Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
	at com.registration.tests.notWorking.RegistrationAllCitiesTest.registrationAllCitiesTest(RegistrationAllCitiesTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationAllCitiesTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.registration.tests.notWorking.RegistrationAllCitiesTest@2bb8ee96]" name="tearDown" is-config="true" duration-ms="7015" started-at="2012-09-04T16:10:45Z" finished-at="2012-09-04T16:10:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.registration.tests.notWorking.RegistrationAllCitiesTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
